src/language-yaml/utils.js

```diff
@@ -256,7 +256,7 @@ function getBlockValueLineContents(
   const leadingSpaceCount =
     node.indent === null
       ? ((match) => (match ? match[1].length : Number.POSITIVE_INFINITY))(
-          content.match(/^( *)\S/m)
+          content.match(/^( *)[^\n\r ]/m)
         )
```

Comment:
These 3 lines are so hard to read... I even paste it in to astexplorer to understand it's an iife...